src/index.ts

```diff
@@ -250,8 +252,12 @@ export async function install(
     logError('no "node_modules" directory exists. Did you run "npm install" first?');
     return;
   }
-
-  const allInstallSpecifiers = new Set(installTargets.map(dep => dep.specifier).sort());
+  const allInstallSpecifiers = new Set(
+    installTargets
+      .map(dep => dep.specifier)
+      .concat(Object.values(aliases))
```

Comment:
This line can be removed and then an entry in `webDependencies` would need to be specified for `aliases` values. Doing this would remove the small overlap between `aliases` and `webDependencies` that is best described as "if specified as alias, also make sure it's used as an installSpecifier", at the cost of a harsher UX.

Reply:
Yea, this is a tough one. For now, let's remove this line so that we can keep the idea of "mark all of your dependencies in webDependences".  This will cause less trouble as we add support for v2 object-syntax of webDependencies, where semver info will be needed. Then we can do a final confirmation/revisit before cutting v2.